src/update-feeds-loops/submit-transactions.test.ts

```diff
@@ -317,7 +493,7 @@ describe(submitTransactionsModule.submitTransaction.name, () => {
     jest.spyOn(gasEstimationModule, 'estimateMulticallGasLimit').mockResolvedValue(BigInt(500_000));
     jest.spyOn(gasPriceModule, 'getRecommendedGasPrice').mockReturnValue(BigInt(100_000_000));
     const api3ServerV1 = generateMockApi3ServerV1();
-    jest.spyOn(api3ServerV1.tryMulticall, 'send').mockReturnValue({ hash: '0xTransactionHash' });
+    jest.spyOn(api3ServerV1, 'tryMulticall').mockReturnValue({ hash: '0xTransactionHash' });
```

Comment:
I just wanted to be consistent with how we call contract methods on all other projects using ethers.js. I'll revert this change because I don't care that much tho